### PR TITLE
Chore template project assignment docs

### PR DESCRIPTION
**Pull request by eranelbaz**
### Issue & Steps to Reproduce / Feature Request
[//]: <> (choose “resolves” “fixes” or “closes” and put link for the issue)
missing docs for template project assignment
### Solution
add it to the template resource docs

### CODE REVIEW

File: examples/resources/env0_template/resource.tf [hunk 1]
```diff
@@ -7,9 +7,14 @@ data "env0_ssh_key" "my_key" {
 }
 
 resource "env0_template" "example" {
-  name        = "example"
+  name = "example"
```
Comment (avnerenv0) [inline examples/resources/env0_template/resource.tf:10]: please keep the good formatting, you can use `terraform fmt` for it

**Review by avnerenv0 — COMMENTED** [review]
What about docs for the `env0_template_project_assignment` resource?

File: examples/resources/env0_template/resource.tf (hunk 1 above)
Comment (eranelbaz) [reply examples/resources/env0_template/resource.tf:10]: 265ba8f6

Comment (avnerenv0) [reply examples/resources/env0_template/resource.tf:10]: that commit is exactly the opposite of what I want :D

Comment (avnerenv0) [reply examples/resources/env0_template/resource.tf:10]: `terraform fmt` will allign all properties to start on the same column

**Comment by eranelbaz** [comment]
@avnerenv0 60f17744

File: examples/resources/env0_template/resource.tf (hunk 1 above)
Comment (eranelbaz) [reply examples/resources/env0_template/resource.tf]: I thought GoLang with terraform plugin does that
56d1379a

File: examples/resources/env0_template_project_assignment/resource.tf [hunk 2]
```diff
@@ -0,0 +1,20 @@
+data "env0_project" "default_project" {
+  name = "Default Organization Project"
+}
+
+data "env0_ssh_key" "my_key" {
```
Comment (avnerenv0) [inline examples/resources/env0_template_project_assignment/resource.tf:5]: please remove the ssh key from here it's not part of this thing

Comment (eranelbaz) [reply examples/resources/env0_template_project_assignment/resource.tf:5]: 747cfb10

Comment (eranelbaz) [reply examples/resources/env0_template_project_assignment/resource.tf]: 939db789

**Review by avnerenv0 — APPROVED** [review]
🥇

**Pull request merged**